dpppt-backend-sdk/dpppt-backend-sdk-ws/src/main/java/org/dpppt/backend/sdk/ws/controller/GaenController.java

```diff
@@ -140,15 +144,27 @@ public GaenController(GAENDataService dataService, FakeKeyService fakeKeyService
 			}
 
 			if (key.getRollingPeriod().equals(0)) {
-				//currently only android seems to send 0 which can never be valid, since a non used key should not be submitted
-				//default value according to EN is 144, so just set it to that. If we ever get 0 from iOS we should log it, since
-				//this should not happen
+				logger.error("RollingPeriod should NOT be 0, fixing it and using 144");
 				key.setRollingPeriod(GaenKey.GaenKeyDefaultRollingPeriod);
-				if (userAgent.toLowerCase().contains("ios")) {
-					logger.error("Received a rolling period of 0 for an iOS User-Agent");
+			}
+			
+			if(delayTodaysKeys) {
+				// Additionally to delaying keys this feature also make sure rolling period is always set to 144 
+				// to make sure iOS 13.5.x does not ignore the TEK.
```

Comment:
There should be some comment here saying that the calculation of the `Rolling Proximity Identifier` on iOS will still be correct, but that it will need to check more keys per day.